bukkit/src/main/java/net/whimxiqal/journey/bukkit/util/MaterialGroups.java

```diff
@@ -223,6 +223,15 @@ private MaterialGroups() {
   public static boolean isVerticallySpecialPassable(Material material) {
     switch (material) {
       case LADDER:
+      case VINE:
+      case SCAFFOLDING:
+      case WATER:
```

Comment:
Traveling through water is handled via a special "swim" mode, so water should be considered impassible normally.

Reply:
Odd that it doesn't seem to be able to go up waterfalls at all normally, bug maybe?

Reply:
Removed these changes as well as they're in `MaterialGroups`, should an issue be made for the water thing though?